spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/groovy/template/GroovyTemplateAvailabilityProvider.java

```diff
@@ -38,11 +38,14 @@ public boolean isTemplateAvailable(String view, Environment environment,
 		if (ClassUtils.isPresent("groovy.text.TemplateEngine", classLoader)) {
 			PropertyResolver resolver = new RelaxedPropertyResolver(environment,
 					"spring.groovy.template.");
+			String loaderPath = resolver.getProperty("resource-loader-path",
+					GroovyTemplateProperties.DEFAULT_RESOURCE_LOADER_PATH);
 			String prefix = resolver.getProperty("prefix",
 					GroovyTemplateProperties.DEFAULT_PREFIX);
 			String suffix = resolver.getProperty("suffix",
 					GroovyTemplateProperties.DEFAULT_SUFFIX);
-			return resourceLoader.getResource(prefix + view + suffix).exists();
+			return resourceLoader.getResource(loaderPath + prefix + view + suffix)
```

Comment:
Please check what is going on here, as it changes the way Groovy templates are detected. As far as I could tell, there was a bug here.